### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by inanimate631**

### CODE REVIEW

File: frontend/src/Routes/start.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { createBasePage } from './base.page';
+
+export function startModue(module: { start: () => void }) {
```
Comment (RetiredHedgehog) [inline frontend/src/Routes/start.ts:3]: Typo, should be `startModule`

File: frontend/src/Pages/index.ts [hunk 2]
```diff
@@ -1,2 +1,16 @@
...
+export function addFunctionalToPage(url: string) {
+  const header = document.querySelector('header') as HTMLElement;
+  header.style.display = 'block';
+  switch (url) {
+    case '':
+      header.style.display = 'none';
+      registrationPageInit();
+      break;
+    case 'autorization':
+      header.style.display = 'none';
```
Comment (Doroshkeviches) [inline frontend/src/Pages/index.ts:12]: case can be grouped like  : 
case ' '
case 'authorization'
//code

**Review by Doroshkeviches — APPROVED** [review]

File: frontend/src/Pages/index.ts [hunk 3]
```diff
@@ -1,2 +1,16 @@
 import { registrationPageInit } from './registrationPage/scripts';
-registrationPageInit();
+
+export function addFunctionalToPage(url: string) {
```
Comment (JaneEllison) [inline frontend/src/Pages/index.ts:3]: A little bit strange name
Mb initPage? 
Or smth like this

File: frontend/src/Routes/module.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  start() {
+    if (this.routes) this.initRoutes();
+  }
+
+  initRoutes() {
+    window.addEventListener('hashchange', this.renderRout.bind(this));
+    window.addEventListener('load', this.renderRout.bind(this));
+  }
+
+  renderRout() {
```
Comment (JaneEllison) [inline frontend/src/Routes/module.ts:19]: Typo 
RenderRoute

Comment (JaneEllison) [reply frontend/src/Routes/module.ts:19]: And mb renderPage suits better here

**Review by JaneEllison — APPROVED** [review]

**Review by RetiredHedgehog — APPROVED** [review]

**Review by RetiredHedgehog — APPROVED** [review]

**Pull request merged**